clients/client-java/src/main/java/org/apache/gravitino/client/HTTPClient.java

```diff
@@ -233,7 +233,9 @@ private void throwFailure(
     }
 
     errorHandler.accept(errorResponse);
-
+    if (response.getCode() == 403) {
```

Comment:
You should add this to client `ErrorHandlers`.

Reply:
fixed